dashboard/src/app/modules/operator/modules/operator-ui/components/operator-form/operator-form.component.ts

```diff
@@ -1,25 +1,23 @@
 import { Component, EventEmitter, Input, OnChanges, OnInit, Output, SimpleChanges } from '@angular/core';
-import { FormBuilder, FormGroup, Validators, AbstractControl } from '@angular/forms';
-import { filter, takeUntil, tap, map, throttleTime } from 'rxjs/operators';
+import { AbstractControl, FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { ToastrService } from 'ngx-toastr';
 import { Subject } from 'rxjs';
-
-import { AuthenticationService } from '~/core/services/authentication/authentication.service';
-import { Address, Bank, Company, Operator } from '~/core/entities/operator/operator';
-import { FormAddress } from '~/shared/modules/form/forms/form-address';
-import { FormCompany } from '~/shared/modules/form/forms/form-company';
-import { FormContact } from '~/shared/modules/form/forms/form-contact';
-import { Contact } from '~/core/entities/shared/contact';
-import { FormBank } from '~/shared/modules/form/forms/form-bank';
-import { bankValidator } from '~/shared/modules/form/validators/bank.validator';
+import { filter, map, takeUntil, tap, throttleTime } from 'rxjs/operators';
 import { DestroyObservable } from '~/core/components/destroy-observable';
-import { Groups } from '~/core/enums/user/groups';
-import { CompanyService } from '~/modules/company/services/company.service';
-import { OperatorStoreService } from '~/modules/operator/services/operator-store.service';
 import { CompanyInterface } from '~/core/entities/api/shared/common/interfaces/CompanyInterface';
+import { Address, Bank, Company, Operator } from '~/core/entities/operator/operator';
+import { Groups } from '~/core/enums/user/groups';
+import { Roles } from '~/core/enums/user/roles';
 import { catchHttpStatus } from '~/core/operators/catchHttpStatus';
+import { AuthenticationService } from '~/core/services/authentication/authentication.service';
+import { CompanyService } from '~/modules/company/services/company.service';
 import { OperatorApiService } from '~/modules/operator/services/operator-api.service';
-import { Roles } from '~/core/enums/user/roles';
+import { OperatorStoreService } from '~/modules/operator/services/operator-store.service';
+import { FormAddress } from '../../../../../../shared/modules/form/forms/form-address';
```

Comment:
ça ne marche pas `~/shared` ?

Reply:
En fait y'a 2 shared ... : /.
Y'a le `~/shared` qui pointe vers les interfaces partagées avec le BE. 
Et faudrait créer un autre shared (`~/dashboard-shared`) qui pointe vers les composants / interfaces partagés coté FE.

Reply:
J'ai un peu envie de l'appeler `common` celui du dashboard pour que ce soit plus clair.